neonvm/controllers/virtualmachine_controller.go

```diff
@@ -364,8 +374,30 @@ func (r *VirtualMachineReconciler) doReconcile(ctx context.Context, virtualmachi
 		vmRunner := &corev1.Pod{}
 		err := r.Get(ctx, types.NamespacedName{Name: virtualmachine.Status.PodName, Namespace: virtualmachine.Namespace}, vmRunner)
 		if err != nil && apierrors.IsNotFound(err) {
+			// Check if the ssh secret already exists, if not create a new one
+			sshSecret := &corev1.Secret{}
+			err := r.Get(ctx, types.NamespacedName{Name: virtualmachine.Status.SSHSecretName, Namespace: virtualmachine.Namespace}, sshSecret)
```

Comment:
I'd define the struct here on separate line, and one line per field. Otherwise, line is too long.

Reply:
resolved in 612191b8cfeff762ae8a499b3340a287521c6251